### DISCUSSION TITLE
Is using Mason on Termux not supported?

### DISCUSSION THREAD

**Discussion by yazigegeda** [mason-org/mason.nvim#1288] [Q&A]
![Screenshot_2023-05-05-08-58-59-53_84d3000e3f4017145260f7618db1d683](https://user-images.githubusercontent.com/63853093/236359366-9c4b94fa-b59c-49a2-9c8c-c2b32483e220.jpg)
![Screenshot_2023-05-05-08-53-41-81_84d3000e3f4017145260f7618db1d683](https://user-images.githubusercontent.com/63853093/236359370-c9c38056-5d79-491b-9b6a-79b01c889da0.jpg)
![Screenshot_2023-05-05-08-52-15-14_84d3000e3f4017145260f7618db1d683](https://user-images.githubusercontent.com/63853093/236359373-ee35f0a5-73a6-4904-b123-3dcfc49cf631.jpg)

Is there a solution available?

**Comment by RiteshChepuri** [comment]
I think there is no support for some language servers on termux. Mason supports on termux , other lsp's and linters are working fine for me.

**Comment by williamboman** [comment]
It depends on the package. Packages provided via platforms such as npm or pypi should generally be installable on Termux, but packages like `lua-language-server` is not. This is due to the fact that binaries that target Bionic libc are virtually never provided upstream. You'll want to use Termux's own package manager for these (it seems to have pretty wide coverage, and they often provide custom patches in order to make them build).

**Reply by gokayburuc** [reply]
I 'm quite new in Neovim. Mason works pretty well in my Ubuntu Desktop but got some issue in my Android tablet. i got same problem in my Android Termux. i installed Lua LSP  via `pkg install lua-language-server` but i don't know how to define custom path to this LSP in Mason. How can i customize paths.

**Comment by DNeir** [comment]
https://github.com/Amirulmuuminin/setup-mason-for-termux ;)
